csunplugged/topics/content/en/curriculum-areas.yaml

```diff
@@ -1,16 +1,72 @@
-maths:
-  name: Maths
+literacy:
+  name: Literacy
   children:
-    maths-geometry:
-      name: Maths - Geometry
-    maths-algebra:
-      name: Maths - Algebra
+    reading:
+      name: Reading
+    writing:
+      name: Writing
 
-science:
-  name: Science
+mathematics:
+  name: Mathematics
+  children:
+    geometry:
+      name: Geometry
+    measurement:
+      name: Measurement
+    numeracy:
+      name: Numeracy
+    algebra:
+      name: Algebra
+
+music:
+  name: Music
 
 art:
   name: Art
 
-design:
-  name: Design
+science:
+  name: Science
+  children:
+    biology:
```

Comment:
it can be whatever you like. I removed the ``science-`` prefix since it's just a duplicate of the parent.